virt/virt-web-console-overview.adoc

```diff
@@ -876,6 +879,36 @@ Click the Options menu {kebab} beside a snapshot to select *Edit labels*, *Edit
 |====
 =====
 
+[id="ui-virtualmachine-details-diagnostic_{context}"]
+==== Diagnostic tab
+
+You can view the status conditions and volume snapshot status on the *Diagnostic* tab.
+
+.Diagnostic tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
+
+|*Status conditions* table
+|List of conditions reported for all aspects of a VM.
+
+|*Filter* field
+|Filter status conditions by category and condition.
```

Comment:
FYI - I recently updated the assembly tables to align with IGS: Sentence fragments are unpunctuated. No periods at the end.